src/components/cards/MetadataCard.jsx

```diff
@@ -1,26 +1,56 @@
 import React from 'react';
+import { range } from 'lodash-es';
+import Skeleton from '@material-ui/lab/Skeleton';
 import List from '@material-ui/core/List';
 import ListItem from '@material-ui/core/ListItem';
 import ListItemIcon from '@material-ui/core/ListItemIcon';
 import ListItemText from '@material-ui/core/ListItemText';
 import IconButton from '@material-ui/core/IconButton';
 import EditIcon from '@material-ui/icons/Edit';
+import Button from '../Button';
 import Text from '../Text';
 import Card from './Card';
 
+const badValues = [null, undefined, ''];
+function fieldValueGood(field) {
+  const value = field?.value;
+  if (badValues.includes(value)) return false;
+  if (value?.time === null && value?.timeSpecificity === null)
+    return false;
+  return true;
+}
+
 export default function MetadataCard({
   title,
   titleId = 'METADATA',
   metadata,
   editable = false,
+  showDefaultValues = false,
+  editButtonId = 'REPORT_METADATA',
   onEdit,
 }) {
+  let metadataToDisplay = metadata;
+  if (!showDefaultValues) {
+    metadataToDisplay = metadata.filter(
+      field =>
+        JSON.stringify(field.value) !==
+        JSON.stringify(field.defaultValue),
+    );
+  }
+
+  metadataToDisplay = metadataToDisplay.filter(
```

Comment:
I think we want to avoid the metadataToDisplay = metadataToDisplay pattern, right? Maybe combine the filters?

Reply:
good call